src/addComponentBoundary.coffee

```diff
@@ -0,0 +1,4 @@
+import addWrapper from './addWrapper'
+
+export default ->
+  addWrapper ({render}) -> render()
```

Comment:
Added this but figure won't expose it until a use case appears (to have the need to manually impose a component boundary)